src/data/modal-instructions.js

```diff
@@ -0,0 +1,22 @@
+/* Onboarding instructions for the Learn More modal */
```

Comment:
I love this! It makes it so much easier to edit the content if we need to without having to comb through the model component's code.